### PR TITLE
Fixed single/double quotes errors in examples

### PR DESCRIPTION
**Pull request by DCSBL**
The example contained a small syntax error, causing my HA installation to yell at me that I did something wrong.

## Proposed change

There are some small errors in the examples, where `'` and `"` are mixed incorrectly.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards]

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/sensor.rest.markdown [hunk 1]
```diff
@@ -280,7 +280,7 @@ sensor:
     sensors:
       date:
         friendly_name: "Date"
-        value_template: "{{ states.sensor.json_time.attributes["date"] }}"
+        value_template: '{{ states.sensor.json_time.attributes["date"] }}'
```
Comment (frenck) [inline source/_integrations/sensor.rest.markdown]:
```suggestion
        value_template: "{{ state_attr('sensor.json_time', 'date') }}"
```

https://developers.home-assistant.io/docs/documenting/yaml-style-guide#quoting-style
https://www.home-assistant.io/docs/configuration/templating/#states

File: source/_integrations/sensor.rest.markdown [hunk 2]
```diff
@@ -324,22 +324,22 @@ sensor:
   - platform: template
     sensors:
       owm_weather:
-        value_template: '{{ state_attr('sensor.owm_report', 'weather')[0]["description"].title() }}'
-        entity_picture_template: '{{ "https://openweathermap.org/img/w/"+state_attr('sensor.owm_report', 'weather')[0]["icon"].lower()+".png" }}'
+        value_template: '{{ state_attr("sensor.owm_report", "weather")[0]["description"].title() }}'
```
Comment (frenck) [inline source/_integrations/sensor.rest.markdown]:
```suggestion
        value_template: "{{ state_attr('sensor.owm_report', 'weather')[0]['description'].title() }}"
```

File: source/_integrations/sensor.rest.markdown [hunk 3]
```diff
@@ -324,22 +324,22 @@ sensor:
...
+        entity_picture_template: '{{ "https://openweathermap.org/img/w/"+state_attr("sensor.owm_report", "weather")[0]["icon"].lower()+".png" }}'
```
Comment (frenck) [inline source/_integrations/sensor.rest.markdown]:
```suggestion
        entity_picture_template: "{{ 'https://openweathermap.org/img/w/' + state_attr('sensor.owm_report', 'weather')[0]['icon'].lower() + '.png' }}"
```

File: source/_integrations/sensor.rest.markdown [hunk 4]
```diff
@@ -324,22 +324,22 @@ sensor:
...
         entity_id: sensor.owm_report
       owm_temp:
         friendly_name: "Outside temp"
-        value_template: '{{ state_attr('sensor.owm_report', 'main')["temp"]-273.15 }}'
+        value_template: '{{ state_attr("sensor.owm_report", "main")["temp"]-273.15 }}'
```
Comment (frenck) [inline source/_integrations/sensor.rest.markdown]:
```suggestion
        value_template: "{{ state_attr(['sensor.owm_report', 'main')['temp'] - 273.15 }}"
```

File: source/_integrations/sensor.rest.markdown [hunk 5]
```diff
@@ -324,22 +324,22 @@ sensor:
...
         unit_of_measurement: "°C"
         entity_id: sensor.owm_report
       owm_pressure:
         friendly_name: "Outside pressure"
-        value_template: '{{ state_attr('sensor.owm_report', 'main')["pressure"] }}'
+        value_template: '{{ state_attr("sensor.owm_report", 'main')["pressure"] }}'
```
Comment (frenck) [inline source/_integrations/sensor.rest.markdown]:
```suggestion
        value_template: "{{ state_attr('sensor.owm_report', 'main')['pressure'] }}"
```

File: source/_integrations/sensor.rest.markdown [hunk 6]
```diff
@@ -324,22 +324,22 @@ sensor:
...
         unit_of_measurement: "hP"
         entity_id: sensor.owm_report
       owm_humidity:
         friendly_name: "Outside humidity"
-        value_template: '{{ state_attr('sensor.owm_report', 'main')["humidity"] }}'
+        value_template: '{{ state_attr("sensor.owm_report", 'main')["humidity"] }}'
```
Comment (frenck) [inline source/_integrations/sensor.rest.markdown]:
```suggestion
        value_template: "{{ state_attr('sensor.owm_report', 'main')['humidity'] }}"
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by DCSBL** [comment]
@frenck, seeing those suggestions, I think we can fix more of those in this page. Doing that now.

File: source/_integrations/sensor.rest.markdown [hunk 7]
```diff
@@ -29,7 +29,7 @@ sensor:
   - platform: rest
     resource: http://IP_ADDRESS/ENDPOINT
     method: POST
-    payload: "{ "device" : "heater" }"
+    payload: '{ "device" : "heater" }'
```
Comment (DCSBL) [inline source/_integrations/sensor.rest.markdown:32]: I am not sure about this one; The JSON wants double quotes.

Comment (frenck) [reply source/_integrations/sensor.rest.markdown:32]: Correct, this is an exceptional case :)

**Review by frenck — APPROVED** [review]
Thanks, @DCSBL 👍

**Pull request merged**